CHANGELOG.adoc

```diff
@@ -31,6 +31,8 @@ toc::[]
 
 === Changed
 
+- auth documentation to reflect latest HC Auth SDK changes
```

Comment:
All the other lines in this file begin with capital letters.

Reply:
👍